services/blockchain-indexer/shared/indexer/indexStatus.js

```diff
@@ -119,7 +119,7 @@ const init = async () => {
 
 	// Index stakers and commission information available in genesis block
 	await indexValidatorCommissionInfo(genesisBlock);
```

Comment:
I think, we can remove the `genesisBlock` as a param here too? We can fetch the `genesisHeight` within the method itself.
```suggestion
	await indexValidatorCommissionInfo();
```